bot.py

```diff
@@ -285,6 +315,41 @@ def help_embed(self, prefix):
             await self.session.post('https://api.modmail.tk/metadata', json=data)
 
             await asyncio.sleep(3600)
+    
+    async def autoupdate_loop(self):
+        while True:
+            if not self.config.get('autoupdates'):
```

Comment:
Just an opinion:

Updating should have a message with the full `CHANGELOG` for that patch sent to them. They would then have to confirm if they want to update.
This is mainly useful for breaking updates (like 2.0) so maybe we only do it on the *y* change? (x.y.z)

Reply:
Add that later, we should use github releases as a changelog.